src/test/java/uk/gov/hmcts/reform/dataextractor/service/impl/BlobServiceImplTest.java

```diff
@@ -0,0 +1,157 @@
+package uk.gov.hmcts.reform.dataextractor.service.impl;
+
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.models.BlobContainerProperties;
+import com.azure.storage.blob.models.BlobHttpHeaders;
+import com.azure.storage.blob.models.ParallelTransferOptions;
+import com.azure.storage.blob.specialized.BlobOutputStream;
+import com.azure.storage.blob.specialized.BlockBlobClient;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import uk.gov.hmcts.reform.dataextractor.model.Output;
+import uk.gov.hmcts.reform.dataextractor.test.utils.Matchers;
+import uk.gov.hmcts.reform.mi.micore.factory.BlobServiceClientFactory;
+
+import java.time.LocalDate;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.isNull;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.dataextractor.service.ContainerConstants.UPDATE_DATE_METADATA;
+import static uk.gov.hmcts.reform.dataextractor.service.impl.BlobServiceImpl.DATE_TIME_FORMATTER;
+
+@ExtendWith(MockitoExtension.class)
+public class BlobServiceImplTest {
+
+    private static final String STORAGE_ACCOUNT = "testStorageAccount";
+    private static final String CLIENT_ID = "testClientId";
+    private static final String CONNECTION_STRING = "testStorageConnectionString";
+    private static final String TEST_CONTAINER_NAME = "testContainer";
+
+
+    private BlobServiceImpl classToTest;
```

Comment:
classToTest always seems to be classToTest = new BlobServiceImpl(CLIENT_ID, CONNECTION_STRING, STORAGE_ACCOUNT, blobServiceClientFactory);

Is there an issue with @InjectMocks or you need to initialise before each test?

Reply:
InjectMocks is not able to inject String values